app/api/answers/submit.go

```diff
@@ -12,10 +12,43 @@ import (
 	"github.com/jinzhu/gorm"
 
 	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/doc"
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 	"github.com/France-ioi/AlgoreaBackend/app/token"
 )
 
+// swagger:operation POST /answers answers items itemGetAnswerToken
+// ---
+// summary: Generate answer token
+// description: Generate a token that can be used by the TaskGrader to ensure task parameters have not been altered.
+//   A task_token has to be given to this service.
+//
+//
+//   This service corresponds to task.php/askValidation <https://github.com/France-ioi/AlgoreaPlatform/blob/master/task/task.php#L202>
```

Comment:
Is this comment future-proof enough? Wasn't this link provided to help you implement it (but so irrelevant in a doc). Should we link to dev doc?

Reply:
fixed